docs/views.md

```diff
@@ -53,3 +53,14 @@ system and can no longer be used to authenticate.
 **Note** It is not recommended to alter the Logout views. They are designed
 specifically for token management, and to respond to Knox authentication.
 Modified forms of the class may cause unpredictable results.
+
+## TokenRefreshView
+Optional view for refreshing the expiration date of a single token.
+It accepts an empty `POST` request against `/refresh` to increase the `expiry` field of the
+token by the amount set in `TOKEN_TTL`.
+This endpoint is only exposed if `AUTO_REFRESH` and `TOKEN_TTL` are enabled. 
```

Comment:
I would argue that the endpoint would still be useful if one want's to disable `AUTO_REFRESH` but still want to be able to manually refresh a token.

I see the point of not enabling the endpoint if `TOKEN_TTL` is `None` as no token would ever expire.
So I would rephrase this a bit and make it clear that we clearly specify that it's only disabled when `TOKEN_TTL` is `None`.

Also see my comment below about this feature.